examples/vpc/resource.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: vpc.tencentcloud.jet.crossplane.io/v1alpha1
+kind: VPC
+metadata:
+  name: example-vpc
+spec:
+  forProvider:
+    cidrBlock: "10.1.0.0/16"
+    name: "test-crossplane"
```

Comment:
Can we skip name and can use metadata.name?

Reply:
yes, but name is required. how to set name with metadata.name and skip name?

Reply:
https://github.com/crossplane/terrajet/blob/main/docs/configuring-a-resource.md#case-1-name-as-external-name-and-terraform-id can you check here ?